tools/msconvert/msconvert_macros.xml

```diff
@@ -25,7 +25,9 @@
     #else
       --input=${input}
       #if hasattr($input, 'display_name')
-      --input_name='${input.display_name}'
+      ##--input_name='${input.display_name}'
+      #set basename = $input.display_name
```

Comment:
I guess the tool is using my local msconvert then, which would explain the difference in ontology version. I see now that the `<sourceFile>` on most of the outputs is also affected by this patch, which explains the other source of problems. Easy to update once the exact syntax of the fix is finalized.